resources/ory/charts/hydra/templates/deployment.yaml

```diff
@@ -37,6 +37,8 @@ spec:
           {{- toYaml . | nindent 8 }}
         {{- end }}
     spec:
+      seccompProfile:
```

Comment:
I think this is not valid here ([reference](https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/pod-v1/#PodSpec)) and should be moved to `spec.securityContext`.